jobs/openshift4/cluster/create/Jenkinsfile

```diff
@@ -17,7 +17,7 @@ node("cirhos_rhel7") {
     timeout(120) { ansiColor("gnome-terminal") { timestamps {
         try {
             currentBuild.displayName = "${currentBuild.displayName} ${clusterName}"
-            currentBuild.description = "clusterName: ${clusterName}\nawsRegion: ${awsRegion}"
+            currentBuild.description = "clusterName: ${clusterName}<br>awsRegion: ${awsRegion}"
```

Comment:
What about adding the kubeadmin password here? Is it a stupid idea?
I closed my original PR since there were some useless changes.

Reply:
probably not a stupid idea, because it is available in the logs anyway...